site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -2135,8 +2135,12 @@ else if (state.getAttribute(STATE_CM_REQUESTED_SECTIONS) != null) {
 				allowUpdateSite = SiteService.allowUpdateSite(site.getId());
 				isMyWorkspace = isSiteMyWorkspace(site);
 				boolean allowUpdateGroupMembership = SiteService.allowUpdateGroupMembership(site.getId());
+				allowViewRoster = SiteService.allowViewRoster(site.getId());
```

Comment:
This line and line 2143 are duplicated from what has been merged in #6597. I think you need to rebase.